### PR TITLE
Added rubygems chapter and toc entry

### PR DESCRIPTION
**Pull request by dulanism**
Trivial: New RubyGems chapter and TOC for 3.1

### CODE REVIEW

File: chapter-rubygems.asciidoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
+[[rubygems]]
+== Node Packaged Modules and npm Registries
```
Comment (DarthHater) [inline chapter-rubygems.asciidoc]: I sell "Ruby Gems and Ruby Gem's based accessories". AKA copy pasta :)

**Comment by mosabua** [comment]
Why not take the rubygenms chapter from master and update accordingly?

**Comment by joedragons** [comment]
Whether you do what @mosabua suggests or just manually move over the contents, from my testing so far I do feel like you'll find a lot of them the same.  I think using existing NX2 stuff will save you time.  2 (more) cents.

**Comment by dulanism** [comment]
Giving it a shot with this tutorial https://ar.al/2891/ (cc @joedragons, @mosabua)

**Pull request merged**